packages/ra-core/src/controller/create/useCreateController.ts

```diff
@@ -175,8 +180,13 @@ export interface CreateControllerProps<RecordType extends RaRecord = RaRecord> {
     resource?: string;
     mutationOptions?: UseMutationOptions<
         RecordType,
+<<<<<<< HEAD
```

Comment:
Some conflict resolution marks are left in your latest commit

Reply:
Yep, saw my mistake: https://github.com/marmelab/react-admin/pull/7698#issuecomment-1131673674

Currently working on it.